server/test/test.js

```diff
@@ -1691,3 +1691,51 @@ describe("User Login API tests:", () => {
     });
   });
 });
+
+//tests on time
+
+const weekPhasesService = require("../services/WeekPhsaseService/weekPhasesService");
+const { WeekPhaseInterval } = require("../services/WeekPhsaseService/weekPhaseInterval");
+const weekPhases = require("../services/WeekPhsaseService/weekPhaseInterval");
+const { WeekPhaseDate } = require("../services/WeekPhsaseService/weekPhaseDate");
+
+
+describe("weekPhase",()=>{
```

Comment:
Stick to our testing convention. these tests are absolutely random until the edge cases are tested and everything is well described.